.github/ISSUE_TEMPLATE/indexer_request.md

```diff
@@ -0,0 +1,17 @@
+---
+name: Indexer Request
+about: Request an indexer for Prowlarr. Duplicated requests or requests covered in existing GitHub Issues will be closed without warning. Please search GitHub prior to requesting.
+title: '(Indexer) '
+labels: 'Type: Indexer Request'
+assignees: ''
+
+---
+
+<!-- Duplicated requests or requests covered in existing GitHub Issues will be closed without warning. Please search GitHub prior to requesting.->
```

Comment:
Can you mention in here to check the Jackett parity issue speifically first before posting a new request.